lld/ELF/Arch/LoongArch.cpp

```diff
@@ -744,6 +784,70 @@ void LoongArch::relocate(uint8_t *loc, const Relocation &rel,
   }
 }
 
+static bool relaxable(ArrayRef<Relocation> relocs, size_t i) {
+  return i + 1 != relocs.size() && relocs[i + 1].type == R_LARCH_RELAX;
```

Comment:
i + 1  ('<=' vs '!=' ?)  relocs.size()

Reply:
> i + 1 ('<=' vs '!=' ?) relocs.size()

In the scenario here, it seems that i+1 < relocs.size() and i+1 ! = relocs.size() are equivalent？